data/data/azure/vnet/internal-lb.tf

```diff
@@ -0,0 +1,68 @@
+resource "azurerm_lb" "internal" {
+  sku                 = "Standard"
+  name                = "${var.cluster_id}-ilb"
+  resource_group_name = "${var.resource_group_name}"
+  location            = "${var.region}"
+
+  frontend_ip_configuration {
+    name      = "internal"
+    subnet_id = "${azurerm_subnet.master_subnet.id}"
+  }
+}
+
+resource "azurerm_lb_backend_address_pool" "master_ilb_pool" {
+  resource_group_name = "${var.resource_group_name}"
+  loadbalancer_id     = "${azurerm_lb.internal.id}"
+  name                = "${var.cluster_id}-ilb-master"
+}
+
+resource "azurerm_lb_rule" "internal_lb_rule_api_internal" {
+  name                           = "api-internal"
```

Comment:
nit: consistent naming scheme is helpful this differs from https://github.com/openshift/installer/pull/1454/files#diff-4fbcc6a2c08d569040d4b49216805509R35

Reply:
fixed in 6901295c0edd936ad95d1643b99e2ae00cf740a7